accepted/future-releases/constructor-tearoffs/feature-specification.md

```diff
@@ -317,6 +317,8 @@ It applies to instance methods as well as local, static and top-level function d
 
 `instanceMethod<int>` (with implicit `this`), `object.instanceMethod<int>` (including `this`) and `super.instanceMethod<int>`.
 
+Cascades can contain explicitly instantiated tearoffs, e.g., `receiver..foo()..instanceMethod<int>..bar`. *Note that this is allowed for consistency, but it will compute a value and discard it, so it is hardly useful.*
```

Comment:
Not necessarily true, since you can call extension members on the function, maybe something like `apply` which calls `Function.apply`:
```dart
foo
  ..bar<int>.apply(argumentList)
  ..bar<String>.apply(otherArgumentList);
```
Probably not going to be widely used, but still potentially useful.

Reply:
Of course! Added a small example to this effect.